react/features/mobile/net-interceptor/functions.js

```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
+import XHRInterceptor from 'react-native/Libraries/Network/XHRInterceptor';
```

Comment:
It's a thing!? Awesome!

Reply:
Yeah, it's what the NetInspector uses :-) Kinda internal, hence the weird import, but if they ever remove it we could vendor it no problem.